### PR TITLE
Development

### PR DESCRIPTION
**Pull request by evaveskova**

### CODE REVIEW

**Comment by Ispirett** [comment]
Hey, @EmirVatric @evaveskova  exceptional work with this project you are doing a fantastic job and your site look gorgeous  :1st_place_medal:   :100: :. There are a few changes that Mircoverse requires before this project can be completed.


#### This project is similar to bootstrap where you create classes to handle the styling and layout of your page ex: **margin-left-5**

#### It will be better to use classes instead of ids throughout this project since you will be creating resuable code.

#### Grid

The CSS files define:

* [ ] A class that represents a row
* [ ] Classes to define columns sized from 1/12th of the row to 100% of the row
* [ ] The defined columns should have breakpoints (example column-large-2)

### Responsiveness

### Breakpoints

* [ ] The CSS files should define a breakpoint for small screens with a width of up to 720px
* [ ] The CSS files should define a breakpoint for medium screens with a width in the range of 720px - 1024px
* [ ] The CSS files should define a breakpoint for large screens with width above 1024px

### Margins

* [ ] The CSS files define separate classes for margin top, bottom, left, right, all margins and auto
* [ ] The margins should span from `0 rem` to `1 rem`
* [ ] The margins should have breakpoints (e.g. margin-sm-top-1)

##### Padding

* [ ] The CSS files define separate classes for padding top, bottom, left, right, all padding and auto
* [ ] The padding should span from `0 rem` to `1 rem`
* [ ] The padding should have breakpoints (example padding-lg-2)

### Positioning

#### Floats

* [ ] The CSS files define separate classes for float left, right and clearing the floats

#### Positions

* [ ] The CSS files define separate classes to change the position property of an element to relative, absolute or fixed

### Colors

#### Background and text

The CSS files define:

* [ ] At least ten different classes to set the background and text colors of an element

### Text

#### Utilities

The CSS files define:

* [ ] Classes to horizontally align text to the left, right or center
* [ ] At least five classes to set the font size of an element
* [ ] Three classes to set the font weight of an element (bold, normal, medium)

#### Border

* [ ] The CSS files define separate classes for border top, bottom, left, right, all border
* [ ] The border should span from 0 to 1rem
* [ ] The border should have classes for making box round
* [ ] The border should have a class defined to transform a box into a circle

**Optional** I suggest using **scss** for this project it makes your life so much easier you won regret it.
This is an article I wrote on *scss*  where it shows how powerful scss is https://medium.com/@i.spirett/how-css-preprocessors-save-you-time-and-money-eaef89f575a3


When finished with these changes please submit for another code review you can use this same pull request link.

**Comment by samgaco** [comment]
Hi @evaveskova and @EmirVatric :hand: 

The project looks amazing guys and everything looks correct, you have updated on almost all the things you were told to add, but it seems like **you still have these points left to cover;**

* [ ] A class that represents a row
* [ ] The CSS files should define a breakpoint for small screens with a width of up to 720px
* [ ] The CSS files should define a breakpoint for medium screens with a width in the range of 720px - 1024px
* [ ] The CSS files should define a breakpoint for large screens with width above 1024px

* **And you have this problem with the logo in the smaller screen size version of the project;**

LEmOUZ

**Commit these changes and submit back for a code review** :muscle: Regards!

**Comment by adriaanbd** [comment]
Working on this Review

File: index.html [hunk 1]
```diff
@@ -1 +1,214 @@
...
+        </div>
+
+        <a href="" class='sing-in padding-2'>SIGN IN</a>
+
+    </nav>
+
+    <main>
+        <section class='flexbox align-items-center center main-felx' id='home'>
+            <h1 class='main-text'>Publish your passions, your way</h1>
+            <h4 class='side-text'>Create a unique and beautiful blog. It’s easy and free.</h5>
```
Comment (adriaanbd) [inline index.html]: Incorrect closing tag for `h4`. You would've noticed if you ran the html file through the required html validation service https://validator.w3.org/

File: custom.scss [hunk 2]
```diff
@@ -0,0 +1,632 @@
...
+        }
+
+    }
+}
+
+@for $i from 1 through 12 {
+    .col-sm-#{$i} {
+        grid-column: span $i;
+
+        @media (max-width: 767.98px) {
```
Comment (adriaanbd) [inline custom.scss]: From requirements:

The CSS files should define a breakpoint for small screens with a width of up to 720px

File: custom.scss [hunk 3]
```diff
@@ -0,0 +1,632 @@
...
+        }
+
+    }
+}
+
+@for $i from 1 through 12 {
+    .col-md-#{$i} {
+        grid-column: span $i;
+
+        @media (max-width: 991.98px) {
```
Comment (adriaanbd) [inline custom.scss]: From requirements:

The CSS files should define a breakpoint for medium screens with a width in the range of 720px - 1024px

File: custom.scss [hunk 4]
```diff
@@ -0,0 +1,632 @@
...
+    .col-#{$i} {
+        grid-column: span $i;
+    }
+}
+
+@for $i from 1 through 12 {
+    .col-lg-#{$i} {
+        grid-column: span $i;
+
+        @media (max-width: 1199.98px) {
```
Comment (adriaanbd) [inline custom.scss]: From requirements:

The CSS files should define a breakpoint for large screens with width above 1024px

**Review by adriaanbd — CHANGES_REQUESTED** [review]
Good job overall, your project looks good, congrats on the effort. Nevertheless, you are still failing to meet the Breakpoint requirements, as mentioned by the previous reviewer. I noticed that you added them, but did not do it with the indicated px number: 

*Breakpoints*

 * The CSS files should define a breakpoint for small screens with a width of up to 720px
 * The CSS files should define a breakpoint for medium screens with a width in the range of 720px - 1024px
* The CSS files should define a breakpoint for large screens with width above 1024px

You're almost done with the HTML/CSS section!

Happy coding :)

**Comment by Ispirett** [comment]
Hey, folks wonderful job with the implementations and changes with this project. :100:  :1st_place_medal: 
also, love the **scss** work
You can go on and merge this project to master.

#### Important before the merge 
- [ ] Add class for display: block and display none

**Don't forget your project completion form**
### new branching rules
Also, an important note for future projects until tick tack toe use a feature branch instead of development you can give that branch descriptive name that describes the feature you are implementing.

**Pull request merged**